src/labels/CustomLabels.labels

```diff
@@ -6547,6 +6725,14 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <!-- {0} = custom label commonPaymentServices i.e. Payment Services -->
         <value>For more information, please see {0} troubleshooting documentation on the Power of Us Hub.</value>
     </labels>
+    <labels>
+        <fullName>geFieldsNotFoundMessage</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Message body when a field or Field Mapping isn't found</shortDescription>
+        <value>The following fields are included in this form, but can't be found: {0}</value>
```

Comment:
@ptewson-sfdo I think you might not need the variable on the end because it'll show up as a bulleted list below this message.